pandas/io/formats/style.py

```diff
@@ -576,10 +576,10 @@ def apply(self, func, axis=0, subset=None, **kwargs):
             on ``axis``), and return an object with the same shape.
             Must return a DataFrame with identical index and
             column labels when ``axis=None``
-        axis : int, str or None
-            apply to each column (``axis=0`` or ``'index'``)
-            or to each row (``axis=1`` or ``'columns'``) or
-            to the entire DataFrame at once with ``axis=None``
+        axis : int, str or None, default 0
```

Comment:
```suggestion
        axis : {0 or ‘index’, 1 or ‘columns’, None}, default 0
```

I suppose the default here is different than using `None` in other places but orthogonal to this, so let's just use same documentation changing None to 0

Reply:
Sorry I don't completely follow this. Your suggestion does not change None to 0. Do you want me to update all the axis docstrings to have this syntax? Or just commit your suggestion for this one.

Reply:
All axis docstrings

Reply:
Done